client/consensus/babe/src/authorship.rs

```diff
@@ -220,7 +219,7 @@ pub fn claim_slot_using_keys(
 					slot_number,
 					&epoch,
 					keys,
-					keystore,
+					keystore.clone(),
```

Comment:
Is this `clone` needed, or would a move do given that it is the last place using the `keystore`?